py-polars/polars/dataframe/frame.py

```diff
@@ -5491,8 +5491,8 @@ def rolling(
         -------
         RollingGroupBy
             Object you can call `.agg` on to aggregate by groups, the result
-            of which will be sorted by `index_column` (but note that if `by` columns are
-            passed, it will only be sorted within each `by` group).
+            of which will be sorted by `index_column` (but note that if `group_by`
+            columns are passed, it will only be sorted within each group).
```

Comment:
I think that the `by` in "...within each `by` group)." isn't needed anymore.
The `group_by` parameter name makes it very obvious what the group is.